features/netsocket/NetworkStack.cpp

```diff
@@ -100,25 +90,13 @@ class NetworkStackWrapper : public NetworkStack
         return address->get_ip_address();
     }
 
-    virtual int gethostbyname(const char *name, SocketAddress *address)
```

Comment:
Does work with LWIP though, because LWIP is providing its own gethostbyname, which does work with AF_UNSPEC.